src/App.jsx

```diff
@@ -11,7 +12,8 @@ function App() {
       <Header />
       <Switch>
         <Route exact path="/" component={Home} />
-        <Route exact path="/Cards/" component={CardDescription} />
+        <Route path="/cards/:needle" component={SearchCards} />
+        <Route path="/Card/" component={CardDescription} />
```

Comment:
why the uppercase for 'Card' ? and the "/" in the end ?

Reply:
Problem resolved!

Reply:
I was asking for camelCase not PascalCase : ).

Reply:
And don't foget to remove the slash after the "/card"

Reply:
/ remove

Reply:
camelCase in the link  is changed. Can you validate your pull request please.